editor/src/clj/editor/git.clj

```diff
@@ -42,16 +42,9 @@
 (defonce required-gitignore-entries ["/.internal"
                                      "/build"])
 
-;; Based on the contents of the .gitignore file we include in template projects.
 (defonce default-gitignore-entries (vec (concat required-gitignore-entries
-                                                [".externalToolBuilders"
-                                                 ".DS_Store"
-                                                 "Thumbs.db"
-                                                 ".lock-wscript"
-                                                 "*.pyc"
-                                                 ".project"
-                                                 ".cproject"
-                                                 "builtins"])))
+                                                [".DS_Store"
```

Comment:
Apart from the `builtins` and possibly the `.cproject`/`.project`, I'm not sure what prompted this change?

E.g. I think no one ever wants to check in "*.pyc" files.
And, given different extensions (and their helper scripts), we might want to exclude `*.class` and `*.o` etc.